plugin/README.md

```diff
@@ -0,0 +1,155 @@
+# Comsos-SDK Plugins
```

Comment:
could this be its own go mod? This would allow for easier release cycles and allow us to tag 1.0 on the streamer sooner. 

If this path is taken I would propose 4 months of v1-alpha and v1 after that so we can get user feedback on things